readthedocs_build/config/test_config.py

```diff
@@ -413,6 +413,70 @@ def it_fails_if_base_does_not_exist(tmpdir):
         assert excinfo.value.code == INVALID_PATH
 
 
+def describe_validate_build():
+
+    def it_fails_if_build_is_invalid_option(tmpdir):
+        apply_fs(tmpdir, minimal_config)
+        build = BuildConfig(
+            {},
+            {'build': {'image': 3.0}},
+            source_file=str(tmpdir.join('readthedocs.yml')),
+            source_position=0)
+        with raises(InvalidConfig) as excinfo:
+            build.validate_build()
+        assert excinfo.value.key == 'build'
```

Comment:
Just curious, why we don't use something like regular `unittest.TestCase` and `assertEqual` and those stuffs in this project?

Reply:
That is just how this code was written originally. _shrug_